### ISSUE TITLE
JavaFX support

### ISSUE THREAD

**Issue by Typhon0** [Thizzer/jtouchbar#9]
Hi, can I use this library with JavaFX ? Thanks

**Comment by ttveldhuis** [comment]
Not at the moment, but I can have support for you before the end of the day if required :)

**Comment by ttveldhuis** [comment]
Added experimental support for JavaFX in the latest snapshot with b6476f3 please let me know if that works for you.

**Comment by Typhon0** [comment]
That's not working for me . I got this error :
```
Error:(26, 18) java: cannot access org.eclipse.swt.widgets.Shell
  class file for org.eclipse.swt.widgets.Shell not found
```

**Comment by ttveldhuis** [comment]
Are you using Maven?

**Comment by Typhon0** [comment]
Yes, I used the build-realease.sh to build the library.

**Comment by ttveldhuis** [comment]
Fixed for 0.2.0 with #28
